### DISCUSSION TITLE
Loki - ad-hoc filter should support metric queries with parsers

### DISCUSSION THREAD

**Discussion by SSvilen** [grafana/grafana#47698] [Data Sources]
Hello,

In PR https://github.com/grafana/grafana/pull/35610 a functionality was added, which allows using of ad-hoc filters with parsers. When parser is used, the filter is added at the end of the query. 
It would be nice, if the same is supported for metric queries. 

For instance, for the following query the ad-hoc filter should be added at the end as a filter.

```
sum by (responsecode) (count_over_time({responsecode="$httpcode"} | regexp "\\/api\\d{0,1}(?P<api>[\\/\\w-\\.]+)" [5m]))
```

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
